lib/mayaUsd/render/MaterialXGenOgsXml/GlslFragmentGenerator.cpp

```diff
@@ -259,44 +259,46 @@ ShaderPtr GlslFragmentGenerator::generate(
     MX_EMIT_INCLUDE(libRoot + "stdlib/genglsl/lib/mx_math.glsl", context, pixelStage);
     emitLineBreak(pixelStage);
 
-    int specularMethod = context.getOptions().hwSpecularEnvironmentMethod;
-    if (specularMethod == SPECULAR_ENVIRONMENT_FIS) {
-        emitLine(
-            "#define DIRECTIONAL_ALBEDO_METHOD "
-                + std::to_string(int(context.getOptions().hwDirectionalAlbedoMethod)),
-            pixelStage,
-            false);
-        emitLineBreak(pixelStage);
-        HwSpecularEnvironmentSamplesPtr pSamples
-            = context.getUserData<HwSpecularEnvironmentSamples>(
-                HwSpecularEnvironmentSamples::name());
-        if (pSamples) {
+    if (lighting) {
```

Comment:
Best seen in "Hide whitespace" mode since this only add 2 lines. Everything else coming from clang-format.